### PR TITLE
Create Carousel package

### PR DESCRIPTION
**Pull request by craigpalermo**
image

### CODE REVIEW

File: packages/carousel/CHANGELOG.json [hunk 1]
```diff
@@ -0,0 +1,567 @@
+{
```
Comment (pcln-james) [inline packages/carousel/CHANGELOG.json:1]: it doesn't seem right for this changelog to have all the changes from DS core

File: packages/carousel/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
+# Change Log - pcln-design-system
```
Comment (pcln-james) [inline packages/carousel/CHANGELOG.md:1]: same here

File: packages/carousel/package.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    "jest": "^26.6.3",
+    "jest-standard-reporter": "^2.0.0",
+    "jest-styled-components": "^6.3.4",
+    "npm-run-all": "^4.1.5",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-test-renderer": "^17.0.2",
+    "rimraf": "^3.0.2",
+    "styled-components": "^4.4.1",
+    "pcln-design-system": "workspace:*"
```
Comment (pcln-james) [inline packages/carousel/package.json:72]: nb: alphabetical order

File: packages/carousel/src/Carousel.spec.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+describe('Carousel', () => {
+  it('should render a carousel', () => {
+    const { getByText, debug } = render(
+      <Carousel>
+        <Flex>Slide 1</Flex>
+        <Flex>Slide 2</Flex>
+        <Flex>Slide 3</Flex>
+      </Carousel>
+    )
+    debug()
```
Comment (pcln-james) [inline packages/carousel/src/Carousel.spec.js:15]: debug

File: packages/carousel/src/index.js [hunk 5]
```diff
@@ -0,0 +1 @@
+export const hello = 1
```
Comment (pcln-james) [inline packages/carousel/src/index.js:1]: don't forget to export the carousel 😛

File: packages/carousel/src/Carousel.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    <CarouselWrapper>
+      <CarouselProvider
+        naturalSlideWidth={100}
+        naturalSlideHeight={100}
+        totalSlides={children.length}
+        visibleSlides={visibleSlides}
+        dragEnabled={false}
+      >
+        <Slider>
+          {children.map((item, index) => {
```
Comment (pcln-james) [inline packages/carousel/src/Carousel.js:34]: should this use React.Children.map instead?

File: packages/carousel/CHANGELOG.json (hunk 1 above)
Comment (craigpalermo) [reply packages/carousel/CHANGELOG.json:1]: My bad, I created the project by copying `core`.

File: packages/carousel/README.md [hunk 7]
```diff
@@ -0,0 +1 @@
+# pcln-carousel
```
Comment (pcln-james) [inline packages/carousel/README.md:1]: nb: documentation for how to use the component

**Review by pcln-james — APPROVED** [review]

File: packages/carousel/README.md (hunk 7 above)
Comment (craigpalermo) [reply packages/carousel/README.md:1]: That'll be included in the next PR to add the rest of the props we discussed.

File: packages/carousel/test/jest.setup.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { ThemeProvider } from 'styled-components'
+
+import '@testing-library/jest-dom/extend-expect'
+import 'jest-styled-components'
+import renderer from 'react-test-renderer'
```
Comment (sdalonzo) [inline packages/carousel/test/jest.setup.js:6]: you don't need this order global.rendererCreateTheme

**Review by sdalonzo — APPROVED** [review]
LGTM, just clean up the unused test-renderer stuff in package.json and jest.setup

**Pull request merged**